ui/pages/v2/KustomizationDetail.tsx

```diff
@@ -0,0 +1,114 @@
+import _ from "lodash";
+import * as React from "react";
+import styled from "styled-components";
+import Flex from "../../components/Flex";
+import KubeStatusIndicator from "../../components/KubeStatusIndicator";
+import Link from "../../components/Link";
+import Page from "../../components/Page";
+import ReconciledObjectsTable from "../../components/ReconciledObjectsTable";
+import Text from "../../components/Text";
+import { useGetKustomization } from "../../hooks/automations";
+import { AutomationKind } from "../../lib/api/core/types.pb";
+import { formatURL } from "../../lib/nav";
+import { V2Routes, WeGONamespace } from "../../lib/types";
+
+type Props = {
+  name: string;
+  className?: string;
+};
+
+const Info = styled.div`
+  padding-bottom: 32px;
+`;
+
+const InfoList = styled(
```

Comment:
I'm seeing this pattern in a couple other pages so we might want to separate this out?

Reply:
It looks like it's in all the different Release Detail and Source Detail pages

Reply:
Can you link to those components? Not seeing them on the `v2` branch.

Reply:
Oh no I'm saying it's in the Figma designs for those pages

Reply:
I typically wait to move stuff until we need to, then the commits make more sense. Good call out though :+1: